src/main.js

```diff
@@ -1,202 +1,35 @@
-    this.intStarted = setInterval(() => {
-      numSec += 1;
-      let minutes = Math.floor(numSec / 60);
-      if (minutes < 10) minutes = `0${minutes}`;
-      let seconds = numSec % 60;
-      if (seconds < 10) seconds = `0${seconds}`;
-      const timerValue = `${minutes}:${seconds}`;
-      pageTimer.firstChild.data = timerValue;
-    }, 1000);
-    this.started = true;
-  }
-
-  stopTimer() {
-    clearInterval(this.intStarted);
-    this.started = false;
-  }
-
-  get isStarted() {
-    return this.started;
-  }
-}
-
-class Game {
-  constructor() {
-    this.imageOnClickBinded = this.imageOnClick.bind(this);
-    this.timer = new Timer();
-    this.fieldIsBlocked = false;
-  }
-
-  appendImages(cards) {
-    this.cards = cards;
-    this.cardsCount = cards.length;
-    this.imagesRandomised = [];
-    this.fillImagesRandom();
-    this.cards.forEach((card, i) => {
-      card.setFrontImage(this.imagesRandomised[i]);
-      card.setBackImage();
-    });
-  }
-
-  start() {
-    this.timer.startTimer();
-  }
-
-  fillImagesRandom() {
-    const imageNumbers = [];
-    for (let i = 0; i < this.cardsCount / 2; i += 1) {
-      imageNumbers.push(i);
-      imageNumbers.push(i);
-    }
-    for (let i = this.cardsCount - 1; i >= 0; i -= 1) {
-      const num = Math.round(Math.random() * i);
-      this.imagesRandomised.push(imageNumbers[num]);
-      imageNumbers.splice(num, 1);
-    }
-  }
-
-  imageOnClick(idClicked) {
-    if (this.fieldIsBlocked) return;
-    switch (this.openedCardsNum) {
-      case 0:
-        this.cards[idClicked].open();
-        return;
-      case 1:
-        if (this.isCardOpened(idClicked)) return;
-        this.cards[idClicked].open();
-        this.fieldIsBlocked = true;
-        this.checkCards();
-        break;
-      default:
-    }
-  }
-
-  isCardOpened(cardId) {
-    const openedCardId = parseInt(this.openedCards[0].div.id.slice(3), 10);
-    return openedCardId === cardId;
-  }
-
-  checkCards() {
-    if (!this.areCardsEqual()) {
-      setTimeout(() => this.returnBackImage(), 1000);
-      return;
-    }
-    setTimeout(() => this.hideGuessedCards(), 600);
-  }
-
-  areCardsEqual() {
-    const hiddenValuesOfOpened = this.openedCards
-      .map((card) => card.hiddenValue);
-    return hiddenValuesOfOpened[0] === hiddenValuesOfOpened[1];
-  }
-
-  returnBackImage() {
-    const { openedCards } = this;
-    openedCards[0].close();
-    openedCards[1].close();
-    this.fieldIsBlocked = false;
-  }
-
-  hideGuessedCards() {
-    const { openedCards } = this;
-    openedCards[0].hide();
-    openedCards[1].hide();
-    this.fieldIsBlocked = false;
-    if (!this.visibleCardsNum) {
-      this.timer.stopTimer();
-    }
-  }
-
-  get openedCardsNum() {
-    return this.openedCards.length;
-  }
-
-  get openedCards() {
-    return this.cards.filter((a) => a.opened);
-  }
-
-  get visibleCardsNum() {
-    return this.cards.filter((a) => a.visible).length;
-  }
-}
+import Game from './GameClass';
```

Comment:
There should be one more file for `DOMManager` class. `main.js` is an entry point